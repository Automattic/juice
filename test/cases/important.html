<html><body>
<a id="a" style="color: blue; background: green;">woot</a>
</body></html>
