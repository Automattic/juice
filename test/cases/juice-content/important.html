<html>
<head>
    <style>
        p {
            color: red !important;
        }
        span {
            color: blue;
        }
        table {
          background-color: black !important;
          width: 100% !important;
        }
    </style>
</head>
<body>
    <p>hi</p>
    <span>there</span>
    <p style="color: blue !important">again</p>
    <table></table>
</body>
</html>
