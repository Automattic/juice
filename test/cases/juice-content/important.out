<html>
<head>
    
</head>
<body>
    <p style="color: red !important;">hi</p>
    <span style="color: blue;">there</span>
    <p style="color: blue !important;">again</p>
    <table style="background-color: black !important; width: 100% !important;" width="100%" bgcolor="black"></table>
</body>
</html>
